test/simctl-e2e-specs.js

```diff
@@ -135,5 +136,16 @@ describe('simctl', function () {
       await setPasteboard(udid, pbContent, encoding);
       (await getPasteboard(udid, encoding)).should.eql(pbContent);
     });
+
+    it('should add media', async function () {
+      const base64Png = 'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNk+M9QDwADhgGAWjR9awAAAABJRU5ErkJggg==';
+      const picturePath = await tempDir.path({prefix: 'pixel', suffix: '.png'});
```

Comment:
This stuff should be in a `before` block, and the cleanup in an `after`.

Reply:
Fixed